py-rattler/rattler/version/version.py

```diff
@@ -54,6 +54,99 @@ def bump(self) -> Version:
         """
         return Version._from_py_version(self._version.bump())
 
+    @property
+    def local(self) -> bool:
+        """
+        Returns true if this version has a local segment defined.
+
+        Examples
+        --------
+        >>> v = Version('1.0')
+        >>> v.local
+        False
+        """
+        return self._version.has_local()
+
+    def as_major_minor(self) -> Optional[Tuple[int, int]]:
+        """
+        Returns the major and minor segments from the version.
```

Comment:
Please also explain that this only works for versions that consist of a major minor version. Otherwise it returns None.